assets/scripts/datalayer/cart-checkout.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+(function ($) {
+  var shopAnalytics = document.shopAnalytics;
+  var $products = $(shopAnalytics.cart.elements.product);
```

Comment:
I would say that if are the products in the cart the variable name should reflect it.